test/Feature/OptionsSpec.hs

```diff
@@ -16,11 +16,40 @@ spec = describe "Allow header" $ do
     r <- request methodOptions "/items" [] ""
     liftIO $
       simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,POST,PATCH,DELETE"
+        matchHeader "Allow" "GET,POST,PUT,PATCH,DELETE,HEAD,OPTIONS"
 
   it "includes read verbs for read-only table" $ do
     r <- request methodOptions "/has_count_column" [] ""
     liftIO $
       simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET"
+        matchHeader "Allow" "GET,HEAD,OPTIONS"
 
+  it "includes read verbs for non auto updatable views" $ do
```

Comment:
I think we need a test for an auto-updatable view, too - otherwise this code-path in the SQL query is not really tested.